res/values/strings_ext.xml

```diff
@@ -14,4 +14,7 @@
     <string name="screenshot_timestamp_exif_title">Save screenshot timestamp to EXIF</string>
     <string name="screenshot_timestamp_exif_summary">Enables adding a timestamp to screenshot EXIF metadata</string>
 
+    <string name="scramble_pin_title">PIN scrambling</string>
+    <string name="scramble_pin_summary">Controls PIN scrambling option when inputting PIN on screen lock.</string>
```

Comment:
Title and summary aren't worded right, I think

Reply:
@muhomorr I copy-pasted from the original values, how should they be worded?

Reply:
Hard to come up with a good one. It's a minor thing, can leave it as is.